### DISCUSSION TITLE
Unable to load addon script from path: 'res://addons/dialogic/plugin.gd'. This might be due to a code error in that script.

### DISCUSSION THREAD

**Discussion by HorgathL** [dialogic-godot/dialogic#1754] [General]
Unable to load addon script from path: 'res://addons/dialogic/plugin.gd'. This might be due to a code error in that script.
Disabling the addon at 'res://addons/dialogic/plugin.cfg' to prevent further errors.

![image](https://github.com/coppolaemilio/dialogic/assets/145518116/1f53e80d-9005-4c5f-9e4b-0fe1a3f086bd)

Can anyone help :'(

**Comment by Jowan-Spooner** [comment]
Seems like you tried to install dialogic 1.5 into godot 4. It is however only compatible with godot 3.x. If you want godot 4 you will have to use dialogic 2 alpha.

**Reply by HorgathL** [reply]
Ahh I see, thanks a lot!
